### DISCUSSION TITLE
How to rewrite the author of changes?

### DISCUSSION THREAD

**Discussion by treeman** [jj-vcs/jj#4070] [Q&A] [answered]
I've made a couple of changes but I realized I had used the wrong email for the commits. I've configured the email so now new commits use the correct one, but how do I change the old commits?

I thought `edit` (and change description slightly) and the automatic rebase would do the job but it doesn't update the user.

**Comment by PhilipMetzger** [comment, marked as answer]
You're searching for `jj describe --reset-author`

**Reply by treeman** [reply]
Perfect, thanks!
